src/styles/_body.scss

```diff
@@ -3,6 +3,6 @@
 body {
   font-family: 'Montserrat', sans-serif;
   font-size: 14px;
```

Comment:
Let's try to use scss variables, so if using this colour, store it as a variable in the variables.scss file and import it here.